R/mod-panel-section.R

```diff
@@ -186,32 +188,32 @@ mod_panel_section_server <- function(input, output, session, synapse, syn, user,
   })
   certified <- check_certified_user(user$ownerId, syn = syn)
   
-  observeEvent(input$submission, {
-    if (!is.null(input$submission) && nchar(input$submission) > 0) {
-      query <- "SELECT * FROM {submissions_table} WHERE form_data_id = {submission_id()}"
-      
-      result <- existing_syn_submission_data()
-      user_name <- existing_syn_submission_username()  
+  observeEvent(c(input$submission, input$refresh_data), {
+    query_trigger(query_trigger() + 1)
+    
```

Comment:
was this query even used anywhere?

Reply:
The query trigger call is basically just to say, re-run the query on refresh to get the submissions data... it's used when we submit already, now its also used when we click refresh.